### ISSUE TITLE
Seller Experience SPIKE: Investigate technical requirements for bundling a theme and plugins at checkout

### ISSUE THREAD

**Issue by blackjackkent** [Automattic/wp-calypso#65100]
As part of our work on onboarding for Sellers, we are going to be allowing a user who selects certain Woo-oriented themes to purchase that theme with a bundled WooCommerce plugin already installed. (See paYKcK-1Hs-p2 )

![image](https://user-images.githubusercontent.com/13437011/176534193-d76c950b-fddb-47ca-9b4b-9f01912ef273.png)

For this issue, we need to investigate what needs to occur on the back end in order to:
a) Identify a theme as requiring this bundling during purchase.
b) Execute this bundling after the checkout process.

Speculations from @ianstewart  on the process that might be involved:

```
Ian’s only-maybe-helpful thoughts on how this should be handled but might be a dumb idea

* Register a new taxonomy in the theme showcase for “plugins”
* Limit it to 1 term for now or document that you should only add 1 term for now
* When we determine that a theme should support or feature a specific plugin then we add a term to that theme post in the showcase that matches the plugin slug like woo-commerce
* The theme showcase will check for terms in that taxonomy in a bunch of places
    * Specifically, for the theme switch flow it’ll show new stuff in the switch modal and turn on the “unlock” this theme flow
* After checkout we then use some of the functionality from the Woo onboarding flow to activate Woo if it’s not activated yet on the Business Plan
```

TLDR this will probably be a two part implementation, with one part being adding an additional label to the theme showcase item and checking for that info when displaying the design picker, and the other being adding functionality to the site creation process to install the plugin as part of the automated setup.

Deliverables for this issue - a P2 post summarizing an intended path forward on implementation here and any potential roadblocks or complications.

**Comment by markbiek** [comment]
First P2 draft here: paYKcK-1R4-p2
